src/solvers/nonlinear_solid_operators.cpp

```diff
@@ -159,14 +159,14 @@ mfem::Operator& NonlinearSolidReducedSystemOperator::GetGradient(const mfem::Vec
 
   // Eliminate the fixed boundary DOFs
   //
-  // This call eliminates the appropriate DOFs in m_jacobian and returns the
+  // This call eliminates the appropriate DOFs in jacobian_ and returns the
   // eliminated DOFs in Je. We don't need this so it gets deleted.
   for (auto& bc : ess_bdr_) {
-    auto Je = std::unique_ptr<mfem::HypreParMatrix>(jacobian_->EliminateRowsCols(bc->true_dofs));
+    auto Je = std::unique_ptr<mfem::HypreParMatrix>(jacobian_->EliminateRowsCols(bc.true_dofs));
   }
   return *jacobian_;
 }
 
 NonlinearSolidReducedSystemOperator::~NonlinearSolidReducedSystemOperator() {}
 
-}  // namespace serac
+}  // namespace serac
```

Comment:
What is the final verdict on EOF newlines? Github yells about it, but I'm not sure if it's really worth worrying about.

Reply:
Linux doesn't consider something a line unless there's a newline at the end, so it's probably advisable.  We should enforce it with CI if `make style` can't fix it.